docs/routers/fritzbox-de.md

```diff
@@ -1,12 +1,12 @@
-Diese Anleitung wurde für FRITZ!OS 07.21 geschrieben, sollte jedoch auch mit anderen Firmware-Versionen funktionieren. Ziel ist es, grundlegende Prinzipien für ein reibungsloses Zusammenspiel zwischen Fritz!Box und Pi-hole zu verdeutlichen.
+Diese Anleitung wurde für FRITZ!OS 07.21 geschrieben, sollte jedoch auch mit anderen Firmware-Versionen funktionieren. Ziel ist, grundlegende Prinzipien für ein reibungsloses Zusammenspiel zwischen Fritz!Box und Pi-hole zu verdeutlichen.
```

Comment:
Ich finde es mit dem "es" eigentlich sprachlich flüssiger.

Reply:
Persönlich mag ich die Phrase nicht; finde das kürzere hier dann schöner. Ich hänge aber nicht an der Änderung :man_shrugging: :wink:

Reply:
Grammatikalisch ist beides richtig, das "es" fügt hier etwas mehr "Abstand" ein und grenzt das Ziel von den Prinzipien besser ab. Wir könnten den Satz auch umstellen, wiederum einmal mit und einmal ohne "es":

```
Es ist das Ziel, grundlegende Prinzipien ... zu verdeutlichen.
Das Ziel ist, grundlegende Prinzipien ... zu verdeutlichen.
Ziel ist, grundlegende Prinzipien ... zu verdeutlichen.
```

Statt "es" ginge auch "hier", z.B.:

```
Das Ziel ist hier, grundlegende Prinzipien ... zu verdeutlichen.
```
alternativ "hierbei", "hiermit", ...

Solange wir uns bei der Formulierung der Dokumentation solchen Fragestellungen ausgesetzt sehen ("High-level"), sehe ich unsere Dokumentation in sehr gutem Zustand. Zumal dieser Satz auch ohne Ersatz komplett gestrichen werden könnte :slightly_smiling_face:

Reply:
Und ich wollte doch eigentlich nur einen Typo korrigieren, der mir beim Lesen und Anwenden aufgefallen ist ;-)
Danke für die ausführliche Antwort, @DL6ER .
Habe die Sätze jetzt getauscht und den Term ganz gestrichen. Das wäre jetzt mein finaler Vorschlag; aber macht's wie ihr's für richtig haltet. Man versteht es ja in allen Varianten :-)

Reply:
Schöne Lösung.